xml_inscriptions/bib_only/MA.Bos.MFA.G.08.288.xml

```diff
@@ -35,12 +34,88 @@
+                        </support>
+                        <condition ana="#fragment.single">
+                           <p> reconstructed from five original marble fragments</p>
+                        </condition>
+                     </supportDesc>
+                     <!--  *************************************  <layoutDesc>  ********************************* -->
+                     <layoutDesc>
+                        <layout columns="0" writtenLines="1">
+                           <p>text runs vertically upward</p>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#inscribed"> </handNote>
+                  </handDesc>
+                  
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p>Youth in profile, left foot advanced, wearing an oliver wreath with fruit
+                           and leaves and holding up two pomegranates in his raised left hand. His
+                           flexed right arm and hand (with wrist muscles preserved) held
+                           an anaryballos suspended from a leather strap.</p>
+                     </decoNote>
+                     <decoNote>
+                        <p>tongues in low relief on the shoulder of the aryballos</p>
+                     </decoNote>
+                     <decoNote>
+                        <p>lotus flowers and buds around the body in low relief on the aryballos</p>
+                     </decoNote>
+                     <decoNote>
+                        <p>crescents in low relief radiating from the bottom of the aryballos </p>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               
+               <!--  *************************************  <history>  *********************************  -->
+               
+               <history>
+                  
+                  <summary>
+                     <rs/>  <!-- content can be "modern" or "modern-text". A blank value indicates that the inscription is ancient.  -->
+                  </summary>
+                  
+                  <origin>
+                     <date evidence="#unspecified" notBefore="-0549" notAfter="-0551">about 550
+                        B.C.</date>
+                     <placeName ref="europe.greece">Thebes, Boiotia </placeName>
+                  </origin>
+                  <provenance type="transferred" subtype="transferred">
+                     <p>acquired in <date when="1900">1900</date>Thebes by Edward Perry Warren, according to his records</p>
```

Comment:
Small typos here; also not by other comments about things like `<persName>` and `<orgName>`